security/pkg/k8s/controller/workloadsecret.go

```diff
@@ -188,6 +197,20 @@ func NewSecretController(ca ca.CertificateAuthority, requireOptIn bool, certTTL
 			UpdateFunc: c.scrtUpdated,
 		})
 
+	namespaceLW := listwatch.MultiNamespaceListerWatcher(namespaces, func(namespace string) cache.ListerWatcher {
+		return &cache.ListWatch{
+			ListFunc: func(options metav1.ListOptions) (runtime.Object, error) {
```

Comment:
question: is it possible to pass a label selector to the NS watcher to return only relevant namespaces (at least when not managing unlabeled namespaces)?